core/domain/stats_domain_test.py

```diff
@@ -458,6 +458,43 @@ def test_to_frontend_dict(self):
         self.assertEqual(
             state_stats.to_frontend_dict(), expected_state_stats_dict)
 
+    def test_clone(self):
+        state_stats = stats_domain.StateStats(0, 10, 0, 4, 0, 18, 0, 7, 2, 0, 2)
+        expected_state_stats = state_stats.clone()
+        self.assertEqual(
+            state_stats.total_answers_count_v1,
+            expected_state_stats.total_answers_count_v1)
+        self.assertEqual(
+            state_stats.total_answers_count_v2,
+            expected_state_stats.total_answers_count_v2)
```

Comment:
Could you use `StateStats.to_dict()` instead? That might make this more readable than all the asserts. Here's the function:

https://github.com/IamEzio/oppia/blob/14cd0187601d941c6e45ebb33391ce07e005cece/core/domain/stats_domain.py#L382

Reply:
Done